source/design-system.js

```diff
@@ -1,17 +1,17 @@
 /**
+ * Design System
  * The kitchen sink of all design components
  */
 
-import breadcrumb from 'atoms/breadcrumb';
-import button from 'atoms/button';
-import image from 'atoms/image';
-import icon from 'atoms/icon';
-import listItem from 'atoms/list-item';
-import card from 'molecules/card';
-import jumbotron from 'molecules/jumbotron';
+import * as breadcrumb from 'atoms/breadcrumb';
+import * as button from 'atoms/button';
+import * as image from 'atoms/image';
+import * as icon from 'atoms/icon';
+import * as listItem from 'atoms/list-item';
+import * as card from 'molecules/card';
+import * as jumbotron from 'molecules/jumbotron';
 
-// Plain ol' object in ES6 object shorthand syntax
-const designSystem = {
```

Comment:
If we're dropping use of `designSystem` here, will we able to update the `ds`' above back to `designSystem` without the console errors you were seeing?

Reply:
CORRECT! I'll make this change.

Reply:
This is updated now at implementation in `app-drupal` and `app-pl`.

Reply:
Woot!